models/sessions/scratch/snowflake/snowplow_web_sessions_this_run.sql

```diff
@@ -292,6 +349,11 @@ select
         {%- for conv_def in var('snowplow__conversion_events') %}
     {{ snowplow_web.get_conversion_columns(conv_def, names_only = true)}}
         {%- endfor %}
+    {% if var('snowplow__total_all_conversions', false) %}
```

Comment:
Should this `if` also require `var('snowplow__conversion_events')` to be populated? Something weird could happen if `snowplow__total_all_conversions` is set to true but `snowplow__conversion_events` isn't specified

Reply:
It's already nested within that if block